File: 프로그래머스/unrated/120956. 옹알이 （1）/옹알이 （1）.py
from itertools import permutations
def solution(babbling):
    answer = 0
    pronounce = ["aya", "ye", "woo", "ma"]
    for i in range(1, len(pronounce)+1):
        for word in babbling:
            if word in map(lambda x: ''.join(x), permutations(pronounce, i)):
                answer += 1
    return answer   

File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.java
class Solution {
    public int solution(String[] babbling) {
        int answer = 0;
        String[] str = {"aya","ye","woo","ma"};
        
        for(int i=0;i<babbling.length;i++){
            for(int j=0;j<str.length;j++){
                babbling[i] = babbling[i].replace(str[j], " ");
            }
            if(babbling[i].trim().length()==0){
                answer++;
            }
        }
        return answer;
    }
}